src/features/customerList/CustomerListContainer.tsx

```diff
@@ -108,7 +121,15 @@ const CustomerListContainer = () => {
 
   const [displayNotification, setDisplayNotification] = useState(false);
 
-  const [debouncedSearchVal] = useDebounce(searchVal, 500, {
+  const debounceWait = [
+    SearchBy.STICKER_NUMBER,
+    SearchBy.STICKER_NUMBER_SEASON,
+    SearchBy.BOAT_REGISTRATION_NUMBER,
+  ].includes(searchBy)
+    ? 1200
+    : 500;
```

Comment:
What are these? They could be named and set in a constant variable. Trivial thing anyway.

Reply:
I'm not sure what to call them :D  - They're the default 500ms and extended 1200ms for slower queries; I guess I can add them as constants. Ideally we can remove them when profile queries get faster.